.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+  - repo: https://github.com/ambv/black
+    rev: 19.3b0
```

Comment:
Can you add a an anchor comment here pointing to dev-requirements.txt too

Reply:
Done.